tests/test_cli.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+import sys
+
+import reportdaily as rd
+
+
+def test_version(capsys):
+    """
+    test if the version is correct 
+    """
+    # given the user use the --version option
+    option = ["--version"]
+    version = rd.__version__
+
+    # when = expect the output of the actual version
+    with pytest.raises(SystemExit):
+        rd.parsecli(option)
```

Comment:
Sorry if I nitpick here again. After thinking about a better name, I think `cliargs` (or just `args`) would be a better name. It's the arguments that you pass to `parsecli`.

```suggestion
    cliargs = ["--version"]
    version = rd.__version__

    # when = expect the output of the actual version
    with pytest.raises(SystemExit):
        rd.parsecli(cliargs)
```